packages/types/src/walletAdapter.ts

```diff
@@ -10,13 +10,14 @@ export type WalletAdapterIcons = {
 export type WalletAdapterData = {
   walletId: string;
   walletName: string;
+  type: string;
```

Comment:
This field should be added to `turbo/generators/wallet/templates/src/index.ts.hbs`
And it would be great to have some explanation and comments about it, how a user should fill this field.